api/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core';
+import { AppModule } from './app.module';
+
+async function bootstrap() {
+  const app = await NestFactory.create(AppModule);
+  await app.listen(8080);
```

Comment:
use `app.listen(process.env.PORT || 8080)` - for deploy should be applied port from process.env